File: sync/v1/sync_map_item/read-query-map/read-query-map.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Sync.V1.Service.SyncMap;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var syncMapItems = SyncMapItemResource.Read(
            from: "steph_curry",
            pathMapSid: "Players",
            order: SyncMapItemResource.QueryResultOrderEnum.Asc,
            pathServiceSid: "ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
        );

        foreach(var record in syncMapItems) {
           Console.WriteLine(record.Key);
        }
    }
}

File: sync/v1/sync_map_item/read-query-map/read-query-map.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

sync_map_items = @client.sync
                        .services('ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
                        .sync_maps('MPXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
                        .sync_map_items
                        .list(from: 'steph_curry', order: 'asc')

sync_map_items.each do |record|
  puts record.key
end
